functions/icons.zsh

```diff
@@ -86,6 +87,76 @@ case $POWERLEVEL9K_MODE in
   'awesome-fontconfig')
     # fontconfig with awesome-font required! See
     # https://github.com/gabrielelana/awesome-terminal-fonts
+    # if not defined, set recommended linux path
+    typeset -p "POWERLEVEL9K_FONTAWESOME_PATH" > /dev/null 2>&1  || POWERLEVEL9K_FONTAWESOME_PATH=~/.fonts
```

Comment:
@gabrielelana - Just checking in on this :) We can't proceed on our end until OSX has the script in the user directory, otherwise we'll break everything for a large number of users.